runtime/common/src/conf/notice/license-mappings.xml

```diff
@@ -0,0 +1,14 @@
+<license-lookup xmlns="https://source.jasig.org/schemas/maven-notice-plugin/license-lookup" >
+    <artifact>
+        <groupId>com.guicedee.services</groupId>
+        <artifactId>commons-lang3</artifactId>
+        <version>1.1.0.7</version>
+        <license>The Apache License, Version 2.0</license>
+    </artifact>
+    <artifact>
+        <groupId>com.guicedee.services</groupId>
+        <artifactId>commons-collections4</artifactId>
+        <version>1.1.0.7</version>
+        <license>The Apache License, Version 2.0</license>
+    </artifact>
+</license-lookup>
```

Comment:
We don't need this file, right?

Reply:
I was half through and hasn't done cleanup just wanted to get your initial feedback with design first.